features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,21 @@
-# put step definitions in this file
\ No newline at end of file
+# put step definitions in this file
+
+When(/^I visit the website$/) do
+  visit ('/')
+end
+
+When(/^see my name$/) do
+  expect(page).to have_content 'Ebba'
+end
```

Comment:
This step def should be  `expect(page).to have_content content`. Your step needs to read like `Then I should see "Ebba"`. Then when you run cucumber, it will give you the step definition with RegEx (the stuff with the $ and all the weird numbers and slashes). Then you can take *any* text input and expect the page to have it. Then you can reuse the step - very DRY.